.github/workflows/ci.yml

```diff
@@ -30,6 +30,15 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 6379:6379
```

Comment:
I don't see where you initialize the workers and the redis server, it can be done in the CI script or in the test

Reply:
Yup, this was the last missing piece.